src/qt/walletmodel.h

```diff
@@ -144,13 +144,15 @@ class WalletModel : public QObject
     // Return status record for SendCoins, contains error id + information
     struct SendCoinsReturn
     {
-        SendCoinsReturn(StatusCode _status = OK, QString _reasonCommitFailed = "")
+        SendCoinsReturn(StatusCode _status = OK, QString _reasonCommitFailed = "", bool _m_fallback_fee_used = false)
             : status(_status),
-              reasonCommitFailed(_reasonCommitFailed)
+              reasonCommitFailed(_reasonCommitFailed),
+              m_fallback_fee_used(_m_fallback_fee_used)
         {
         }
         StatusCode status;
         QString reasonCommitFailed;
+        bool m_fallback_fee_used;
```

Comment:
These should be const?

Reply:
I keep it non const for now to avoid changing code that receives that struct, also, other variables are also non-const.